### DISCUSSION TITLE
Add option for custom title separator

### DISCUSSION THREAD

**Discussion by aerni** [aerni/statamic-advanced-seo#75] [Ideas]
It might be handy to be able to set a custom title separator in the general site defaults in addition to the pre-defined options.

It probably doesn't make sense to make this configurable on the content defaults and each entry/term, though. If you want to go custom on an entry/term, you can always disable the site name and write a fully custom meta title.

**Comment by daun** [comment]
I would like to voice my approval of this suggested feature :) We usually use the long em dash (—) as title separator, however the addon unfortunately only allows the normal short dash (-) in the predefined list of separators. I've been able to set this to the long dash manually inside the yaml file, but for some reason this is no longer possible after a recent update.

I would suggest adding an array config item like `advanced-seo.additional_title_separators` that gets appended to the preset.

```php
return [
    'additional_title_separators' => ['—'] // long dash
];
```

**Comment by aerni** [comment]
This idea was implemented in https://github.com/aerni/statamic-advanced-seo/pull/182. It was actually pretty straight-forward as the select fieldtype allows for custom options already. I also added the long dash to the available default options.
